src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -50,6 +69,14 @@ protected void doFilterInternal(final HttpServletRequest request,
                     timing,
                     contentEncoding,
                     acceptEncoding);
+
+            nakadiKpiPublisher.publish(accessLogEventType, () -> new JSONObject()
+                    .put("method", method)
+                    .put("path", path)
+                    .put("query", query)
+                    .put("app", Hex.encodeHexString(sha256MessageDigest.digest(user.getBytes(Charsets.UTF_8))))
```

Comment:
I would add both original and hashed app name. One we need for audit and some specific cases of KPI later,
and hash we need for the count of unique users KPI right now.

Reply:
I would also add a salt